.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -32,7 +32,7 @@ Other classifications:
 
 > 0.2.x
 > 0.3.x
-> HEAD
+> Git SHA
```

Comment:
It might be useful to instead have users run `git describe --dirty` which should output something like:

`v0.7.0-32-g8593e2eb-dirty`

This helps understand the exact commit, if their workspace is clean, and understand where it is since the last release at a glance.

Reply:
Good idea :)